mlir/lib/Dialect/Affine/Transforms/LoopTiling.cpp

```diff
@@ -237,6 +237,13 @@ void LoopTiling::getTileSizes(ArrayRef<AffineForOp> band,
 }
 
 void LoopTiling::runOnOperation() {
+  if (cacheSizeInKiB <= 0) {
+    Builder b = Builder(getOperation()->getContext());
```

Comment:
Why did you need to create a builder here? Also, just `&getContext()` will work if you need the context.